atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/backend/KeyValueServiceInstrumentation.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Set;
 import java.util.TreeMap;
 
+@SuppressWarnings("ClassInitializationDeadlock")
 public abstract class KeyValueServiceInstrumentation {
```

Comment:
The CassandraKeyValueServiceInstrumentation is being used in tests in a different package and can not be made package private, would appreciate alternative ideas for fixing the ClassInitDeadlock issue here.

Reply:
You can still make the generated code package private, and provide a builder in this class to avoid the issue outside of the package

Reply:
This is only used in benchmarking, and not worth the effort to clean it up